GitWorkflow/vscode-app/Readme.md

```diff
@@ -0,0 +1,89 @@
+# VSCode Application for SAP Data Intelligence
+
+The VSCode application is a [Visual Studio Code](https://code.visualstudio.com/) extension for
```

Comment:
The distinction and (legal) relationship between this DI app and the standalone PC product should be made clearer. Especially it should be clarified who (SAP or MS) provides exactly what at which SLA. Or is this clarified elsewhere, if so where?

Reply:
Added disclaimer to the vscode readme